### PR TITLE
 ci: bindings.yml: use 'build' rather than 'check' naming 

### PR DESCRIPTION
**Pull request by istankovic**

### CODE REVIEW

File: .github/workflows/bindings.yml [hunk 1]
```diff
@@ -52,19 +52,28 @@ jobs:
...
-      - uses: actions-rust-lang/setup-rust-toolchain@v1  # this implicitly caches Rust tools and build artifacts
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
         with:
           rustflags: ''
-          target: "aarch64-apple-ios,x86_64-apple-ios,aarch64-apple-ios-sim"
+          target: ${{ matrix.task.target }}
       - name: Setup cargo-make
         uses: davidB/rust-cargo-make@v1
       - run: |
           cd crypto-ffi
```
Comment (SimonThormeyer) [inline .github/workflows/bindings.yml:75]: Maybe we can add a `- name` here too? I find it confusing that in the overview this is called `cd crypto-ffi`

Comment (istankovic) [reply .github/workflows/bindings.yml:75]: Totally. :+1:  :-)

File: .github/workflows/bindings.yml [hunk 2]
```diff
@@ -52,19 +52,28 @@ jobs:
...
-          cargo make ios-create-xcframework
```
Comment (SimonThormeyer) [inline .github/workflows/bindings.yml:67]: Do we want a downstream job that does `cargo make ios-create-xcframework` and
```
- needs: [ios-device, ios-simulator-x86, ios-simulator-arm]
```
?

Comment (istankovic) [reply .github/workflows/bindings.yml:67]: Yeah, there's going to be another job that collects all of the artifacts. Note this is still a draft. :wink:

File: .github/workflows/bindings.yml (hunk 1 above)
Comment (istankovic) [reply .github/workflows/bindings.yml:67]: Done.

File: .github/workflows/bindings.yml [hunk 3]
```diff
@@ -62,11 +62,12 @@ jobs:
...
       - name: Setup cargo-make
         uses: davidB/rust-cargo-make@v1
-      - run: |
+      - name: Build Swift package
+        run: |
           cd crypto-ffi
           cargo make ios-create-xcframework
 
-  check-wasm:
+  build-wasm:
```
Comment (SimonThormeyer) [inline .github/workflows/bindings.yml:70]: This could be named `build-and-test-wasm`, note `bun test` below

**Review by SimonThormeyer — APPROVED** [review]
Approved, just one minor thing regarding the wasm step

**Pull request merged**